2-RTB/Piano_di_qualifica/cruscotto.tex

```diff
@@ -1 +1,142 @@
-\chapter{Cruscotto di controllo della qualità}
\ No newline at end of file
+\chapter{Cruscotto di controllo della qualità} \label{sec:cruscotto}
+
+\section{Varianza dell'impegno orario}
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.8\linewidth]{VarOre.png}
+    \caption{Varianza dell'impegno orario per sprint}
+\end{figure}
+\subsection{Analisi}
+Analizzando i valori riportati nel grafico, è riscontrabile una difficoltà nel produrre preventivi orari vicini al consuntivato.\\
+In particolare, viene registrato un valore di poco inferiore all'accettabile nel terzo sprint. Il motivo del numero inferiore di ore produttive è stato esaminato nell'analisi retrospettiva presente nel Piano\_di\_progetto\_v1.0.
+
+\section{Varianza di Budget}
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.8\linewidth]{VarBud.png}
+    \caption{Varianza di Budget per sprint}
+\end{figure}
+\subsection{Analisi}
+Dall'analisi del grafico è immediato osservare che i valori di varianza di badget sono sempre rimasti all'interno della soglia di accettabilità.\\
+È il risultato evidenziato è tuttavia migliorabile: seppur non è mai stato speso di del preventivato in uno sprint, è importante in futuro migliorare la fase di pianificazione. L'obiettivo deve essere quello di produrre preventivi sempre più vicini a quello poi consuntivato.
```

Comment:
Ehhhh spe, possiamo riscriverla meglio, tipo "Il risultato evidenziato è tuttavia migliorabile: sebbene non sia mai stato speso più di quanto preventivato in uno sprint, è importante in futuro migliorare la fase di pianificazione. L'obiettivo deve essere quello di definire preventivi sempre più realistici partendo da quanto consuntivato negli sprint precendenti." 
Poi il dubbio mi resta, sono i preventivi da migliorare o i consuntivi e quindi quanto lavoriamo?